xml/schema/CRM/Aip/AipErrorLog.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+<table>
+  <base>CRM/Aip</base>
+  <class>AipErrorLog</class>
+  <name>civicrm_aip_error_log</name>
+  <comment>AIP Process</comment>
```

Comment:
Seems to be copied from `AipProcess.xml`...

Reply:
Yes, thanks